terminal/src/remote.rs

```diff
@@ -0,0 +1,91 @@
+use std::collections::HashSet;
+
+use librad::git::storage::Storage;
+use librad::git::Urn;
+
+use radicle_common::{git, project};
+
+use crate as term;
+
+fn find_remote(
+    name: &str,
+    storage: &Storage,
+    repo: &git::Repository,
+    urn: &Urn,
+) -> anyhow::Result<Option<String>> {
+    if let Ok(peer_) = name.parse() {
+        // by Peer ID
+        for (name, peer) in git::remotes(repo)? {
+            if peer == peer_ {
+                return Ok(Some(name));
+            }
+        }
+        return Ok(None);
+    }
+
+    // by person's name
+    for (name, peer) in git::remotes(repo)? {
+        if let Some(person) = project::person(&storage, urn.clone(), &peer)? {
+            if person.subject().name.to_string() == *name {
+                return Ok(Some(name));
+            }
+        }
+    }
+    Ok(None)
+}
```

Comment:
Can this function maybe go in `common/project`?

Reply:
done